ledger-service/lf-value-json/src/main/scala/com/digitalasset/daml/lf/value/json/ApiCodecCompressed.scala

```diff
@@ -198,9 +243,7 @@ abstract class ApiCodecCompressed[Cid](
       case prim: Model.DamlLfTypePrim =>
         jsValueToApiPrimitive(value, prim, defs)
       case typeCon: Model.DamlLfTypeCon =>
-        val id = Model.DamlLfIdentifier(
-          typeCon.name.identifier.packageId,
-          typeCon.name.identifier.qualifiedName)
+        val id = typeCon.name.identifier
         // val dt = typeCon.instantiate(defs(id).getOrElse(deserializationError(s"Type $id not found")))
         val dt = Model.damlLfInstantiate(
```

Comment:
Unrelated to this commit: AFAIR `Model.damlLfInstantiate` is a copy of `typeCon.instantiate`, introduced when the daml.lf version had bug and Navigator could not upgrade to the latest daml.lf library version.

New issue for fixing this: https://github.com/digital-asset/daml/issues/2506